README.md

```diff
@@ -103,7 +122,14 @@ Using the object literal notation, Define an object called student and give it t
 Your Answer:
 ```js
 // write code here
-```
+```var student={
```

Comment:
Almost! Key/value pairs in an object end with a `,`, not a `;`, and the syntax for adding functions [is different](https://github.com/ga-wdi-quizzes/week-02/blob/solution/README.md#question-9).